proof/invariant-abstract/ARM/ArchRetype_AI.thy

```diff
@@ -646,8 +646,7 @@ lemma valid_untyped_helper [Retype_AI_assms]:
   have obj_at_pres: "\<And>P x. obj_at P x s \<Longrightarrow> obj_at P x ?ns"
   by (clarsimp simp: obj_at_def dest: domI)
    (erule pspace_no_overlapC [OF pn _ _ cover vp])
-  note blah[simp del] = atLeastAtMost_iff atLeastatMost_subset_iff atLeastLessThan_iff
-        Int_atLeastAtMost atLeastatMost_empty_iff
+  note blah[simp del] = atLeastAtMost_simps
```

Comment:
You don't have to change it in this iteration, but if we don't make reference to the name `blah` further down, it is actually Ok to not have a name here, because the scope is global.

```isabelle
note [simp del] = atLeastAtMost_simps
```

would be legal syntax and Ok here (we just don't want un-named global simp lemmas)

Reply:
Oh yeah, I said a similar thing in another comment, sorry for the dupe. Agreed.